ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/aaa.yml

```diff
@@ -60,7 +60,11 @@ aaa_authorization:
   serial_console: true
   commands:
     all_default: group aaaAuth
-
+    privilege:
+      - level: "10,15"
+        group: "tacacs+ local"
+      - level: "5"
+        group: "RADIUS"
```

Comment:
This should be radius not caps.

![image](https://user-images.githubusercontent.com/88734745/145648193-852c8351-71b1-46af-895a-8714f271b140.png)